data/persons.txt

```diff
@@ -1698,3 +1698,124 @@ effect "fusion cannon cloud"
 	"random angle" 360
 	"random velocity" 10
 	"velocity scale" .2
+
+
+
+person "MasterOfGrey"
+	government "Author"
+	frequency 1000
+	personality
+		heroic plunders opportunistic
+	system
+		government "Hai"
+	phrase
+		word
+			"Why is everything Alphas?"
+			"You're only a terrorist if the people experience terror."
+			"Have you finished the Free Worlds storyline yet?"
+			"Things don’t have to mimic an Earth creature to be not very alien."
+			"I have questions. Questions that do not yet need answers."
+			"There are no bugs in my code, only Xapleaux precursors."
+			"If I were green I would photosynthesize."
+			`"Trust the process" isn't a joke for no reason.`
+			"Have you reviewed Hai Reveal yet?"
+			"I gotta go get ready for work. Play nice and don’t @ me."
+			"I don’t even like yellow."
+			"I really enjoy the backstory for the Marauders."
+			"Falling into the Ember Waste and emerging into absolute silence for the first time in dozens of gameplay hours was profoundly moving."
+			"Planet descriptions will be the death of me."
+			"Have you tried Quyykk's editor for map work?"
+			"Did systems get made big yet?"
+			"Heat lasers and hull lasers are different metas and shouldn't be combined."
+			"<placeholder text>"
+	ship "Modified Ladybug" "Revealer of Hai"
+		"never disabled"
+
+ship "Modified Ladybug"
+	sprite "ship/modified ladybug"
+	thumbnail "ship/modified ladybug"
+	attributes
+		category "Medium Warship"
+		"cost" 8950000
+		
+		# To be buffed:
+		"shields" 26000
+		"hull" 20000
+		
+		"required crew" 16
+		"bunks" 33
+		"mass" 280
+		"drag" 5.4
+		"heat dissipation" .85
+		"fuel capacity" 900
+		"cargo space" 91
+		"outfit space" 611
+		"weapon capacity" 206
+		"engine capacity" 153
+
+		# Features from original Ladybug:
+		"ramscoop" 0.5
+		"cargo scan power" 5
+		"cargo scan speed" 1
+		"outfit scan power" 17
+		"outfit scan speed" 1
+		"tactical scan power" 25
+		"atmosphere scan" 100
+		"asteroid scan power" 30
+		
+		weapon
+			"blast radius" 100
+			"shield damage" 1000
+			"hull damage" 500
+			"hit force" 1500
+	outfits
+		# Offensive turrets will be replaced with better pulse turrets in the future.
+		"Bombardment Turret"
+		"Thermal Repeater Turret" 2
+		"Point Defense Turret"
+		"Warder Anti-Missile"
+
+		"Hai Diamond Regenerator"
+		"Hai Williwaw Cooling" 2
+		"Hai Ravine Batteries"
+		"Double Plasma Core"
+		"Large Heat Shunt"
```

Comment:
```suggestion
		"Systems Core (Medium)"
		"S-970 Regenerator"
		"Crystal Capacitor"
		"Double Plasma Core"
		"Small Heat Shunt" 2
		"Large Heat Shunt"
		"Fuel Processor"
		"Quantum Keystone"
```
+16 outfit capacity and a little more reflective of my actual outfitting approach